### DISCUSSION TITLE
Choice type Field

### DISCUSSION THREAD

**Discussion by arthuroliveira** [strapi/strapi#8802] [Legacy Topics]
Sorry if this is too basic of a question, but I'm new to Strapi, and I was wondering if I could create a field of Choice type. to give the user a limited amout of choice. I know i can do this with relationship, but some choices can be very small limited, I'd rather not have to create another table for it. A goof example would be a Gender field , where the user would be able to select Male and Female. 

Is this possible on Strapi? 

Thank you

**Comment by Yigitali97** [comment]
@arthuroliveira Sure, it is possible. You can use Enum Field type

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
